share/rocm/cmake/ROCMCreatePackage.cmake

```diff
@@ -64,6 +50,27 @@ macro(rocm_create_package)
     set(CPACK_RPM_PACKAGE_AUTOREQPROV
         Off
         CACHE BOOL "turns off rpm autoreqprov field; packages explicity list dependencies")
+    set(CPACK_RPM_FILE_NAME "RPM-DEFAULT")
+
+    rocm_get_debian_version_info(DEBIAN_VERSION)
+    if(DEBIAN_VERSION STREQUAL "unknown")
```

Comment:
Why cant this just be `if(DEFINED ENV{CPACK_DEBIAN_PACKAGE_RELEASE})` instead? And then remove these `rocm_get_debian_version_info` functions.

Reply:
done